lib/stacker/cli.rb

```diff
@@ -111,18 +115,6 @@ def dump stack_name = nil
       end
     end
 
-    desc "fmt [STACK_NAME]", "Re-format template JSON"
-    def fmt stack_name = nil
```

Comment:
@jfhamlin ah, news to me! I'm not opposed to YAML if it can be used on both ends. Really good to know. Some way to convert existing templates to yml would be pretty great. Maybe downloading the remote version (dump) is enough. Not sure about YAML, but AWS seems to reformat the JSON some on upload.